airbyte-integrations/connectors/source-dockerhub/sample_files/invalid_config.json

```diff
@@ -0,0 +1 @@
+{"pokemon_name": "chikapu"}
```

Comment:
{
  "invalid_username": "this should error because wrong name of field supplied"
}